app/components/provider_interface/conditions_component.html.erb

```diff
@@ -1 +1,23 @@
-<%= render SummaryListComponent.new(rows: condition_rows) %>
+<% if condition_rows %>
```

Comment:
Minor: you can get rid of this conditional by implementing `render?` in the component:

https://github.com/github/actionview-component/#conditional-rendering

Reply:
Cool. I'll remove the defensive `&.` calls but implement `#render?` checking the application_choice is in the appropriate state.

Reply:
I think it should bail if the choice is in the wrong state as this means we have a bug in our calling code. It would be better to fail loudly than render nothing and us not know it was broken